test/CMakeLists.txt

```diff
@@ -127,11 +127,18 @@ foreach(file ${files})
     )
     set_tests_properties("${testcase}_default" PROPERTIES LABELS "default")
 
-    add_test(NAME "${testcase}_all"
-      COMMAND ${testcase} ${DOCTEST_TEST_FILTER} --no-skip
-      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-    )
-    set_tests_properties("${testcase}_all" PROPERTIES LABELS "all")
+    # Skip test-unicode_all on MSVC Debug build
```

Comment:
Let's not do this in the CMake file, but rather in the AppVeyor YAML using the ctest command line options.

Reply:
Totally agree and done.

Sorry about that. I wasn't aware of `--exclude-regex` option for `ctest`. Hope this sorts it out.